test/integration/specs/kbaseCellToolbarMenu_test.js

```diff
@@ -0,0 +1,327 @@
+/*global describe, it, browser, afterEach, beforeEach*/
+/* eslint {strict: ['error', 'global']} */
+'use strict';
+
+const {login, openNarrative, clickWhenReady} = require('../wdioUtils.js');
+
+/*
+Notes:
+
+The test narrative has 5 markdown cells. The first cell has summary text, the remaining 4 contain the text "Cell #".
+The first three cells are collapsed, the last two are expanded.
+
+Tests currently cover common cases of cell movement; additional cases can be added, e.g. try to move 
+past the end, or before the beginning, try moving the same cell multiple times, inspect the state
+of all cells after a movement.
+
+This test suite could be expanded to cover all usage of the cell toolbar (kbaseCellToolbarMenu.js), or they
+could be in separate files.
+
+*/
+
+const testData = {
+    common: {
+        unselectedBorder: '1px 1px 1px 5px solid rgb(204, 204, 204)',
+        selectedBorder: '1px 1px 1px 5px solid rgb(75, 184, 86)',
+    },
+    all: {
+        TEST_CASE1: {
```

Comment:
It's a bit difficult to read the number with the string written like this -- could you change it to either `testCase1` or `TEST_CASE_1`, so the digit is clearer?

Reply:
okay, nice.
btw, I'm also applying changes like this which affect integration test  style of the (two) other existing integration test files.

Reply:
Is addressed in an update to the PR.